src/types/Tracking.ts

```diff
@@ -10,7 +10,7 @@ export type PlantingSiteSearchResult = {
   name: string;
   numPlantingZones: number;
   numPlantingSubzones: number;
-  'totalPlants(raw)': number;
+  totalPlants: string;
```

Comment:
using string value so it comes pre-formatted/localized.. we won't be using raw numbers for math